modules/kube_cluster/outputs.tf

```diff
@@ -0,0 +1,34 @@
+output "kube_api_ip" {
+  value       = local.api_vip
+  description = "K8s API IPs"
+}
+
+output "rancher_address" {
+  value       = var.rancher_flavor != "" ? "https://rancher.${local.ingress_ip}.sslip.io" : null
+  description = "Rancher URL"
+}
+
+output "rancher_password" {
```

Comment:
```suggestion
output "rancher_password" {
  sensitive = true
```

Reply:
This may need a `terraform fmt -recursive`

Reply:
I set the rancher password as sensitive but then it required all the dependents to be sensitive as well... So I removed it because I thought the whole output would be hidden.
Also, it looks like `for_each` doesn't like sensitive values either 😅 

```
Plan: 1 to add, 0 to change, 0 to destroy.
╷
│ Error: Invalid for_each argument
│ 
│   on ../../main.tf line 14, in module "kube_cluster":
│   14:   for_each = { for cluster in var.clusters : cluster.name => cluster }
│     ├────────────────
│     │ var.clusters has a sensitive value
│ 
│ Sensitive values, or values derived from sensitive values, cannot be used as for_each arguments. If used, the sensitive value could be exposed as a resource instance key.
```

Reply:
If I use `nonsensitive` there and keep adding sensitive values to the ones complaining, the output looks like:

```
Changes to Outputs:
  + clusters_output = (sensitive value)
```

Reply:
See the updated PR with an alternative solution.